.github/workflows/ci.yml

```diff
@@ -89,7 +89,7 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: 3.7
-      - run: cd python && pip install .
+      - run: cd python && pip install -e mcap
```

Comment:
Maybe this pip install should be replaced by `pipenv install` like we have in the other python workflow?

Reply:
I actually prefer this way because it tests that our `mcap` library properly specifies its dependencies in `setup.cfg`.